README.md

```diff
@@ -1,3 +1,81 @@
 # SUSE Application Collection
 
-Application Collection extension assets and community feedback.
+This extension will help you installing, managing and uninstalling workloads from Application Collection.
+
+This project is composed of:
+
+* The [ui](./ui): a [React](https://es.react.dev/) app that handles the main user interation.
+* A [backend](./backend): an [express](https://expressjs.com/) server that extends the features of the frontend.
+
+## Table of Contents
+
+* [Getting started](#getting-started)
+* [Debug with Docker Desktop](#debug-with-docker-desktop)
+* [Debug with Rancher Desktop](#debug-with-rancher-desktop)
+
```

Comment:
I would remove this section as the table of contents (or outline) will be automatically generated by GitHub and we can forget about it on new updates

```suggestion
```